ani-cli

```diff
@@ -382,6 +397,14 @@ open_selection() {
 	# opens selected episodes one-by-one
 	for ep in $episodes
 	do
+		whether_half="$(echo "$ep" | cut -c1-1)"
+		if [ "$whether_half" = "h" ]
```

Comment:
if you want to compare the first character using shell then do this instead
```sh
 if [ ${str#h} != ${str} ]; then
    # it is half episode
    ... # do stuff here
else
    # it is not half ep
    ... # do stuff here
fi
 ```
basically if the string remains same after removing prefix h (if exists)

 
 > PS. Try not to use echo, it doesn't have concrete specifications, different shells act differently with it

Reply:
reason for doing so in the if statement is to avoid subshells and external utils when something could be done in pure sh.